input/resources/StructureDefinition-profile-SignalHealthcareService.json

```diff
@@ -12,6 +12,14 @@
   "derivation": "constraint",
   "differential": {
     "element": [
+      {
+        "id": "HealthcareService.category",
+        "path": "HealthcareService.category",
+        "binding": {
+          "strength": "example",
```

Comment:
Change strength to 'extensible' at least, but it is 'required' for our model in all situations, isn't it?

Reply:
Updated the Binding Strength as required for "StructureDefinition-profile-SignalHealthcareService". 

https://github.com/enjoysparkling/signal-mso-fhir-profiles/commit/b6d358cc8c9ab6085bbed60573d4f7689d053a65